packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -188,12 +188,21 @@ class CupertinoPageRoute<T> extends PageRoute<T> {
 
   /// True if a Cupertino pop gesture is currently underway for [route].
   ///
+  /// This returns true for both the top route and the bottom route of a
```

Comment:
This is still confusing since exactly what "pop gesture" means isn't self evident. Since we're just returning the value of `route.navigator.userGestureInProgress` you could just say as much and provide an explanation there.

Reply:
done